[functorch] classify some more test failures

zou3519 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #83520
* #83518
* #83517

Classifies test failures for test_vmapvjp and test_vmapjvpall

Test Plan:
- tests

zou3519 committed:
[functorch] classify some more test failures

Classifies test failures for test_vmapvjp and test_vmapjvpall

Test Plan:
- tests

[ghstack-poisoned]

samdow reviewed (approved):
The ending `# ----------------------------` really helps. I was thinking about how we're confusingly grouping tests earlier and this is right

zou3519 committed:
Update on "[functorch] classify some more test failures"


Classifies test failures for test_vmapvjp and test_vmapjvpall

Test Plan:
- tests

[ghstack-poisoned]

zou3519 commented:
@pytorchbot merge -f "functorch CI passed"
